app/_src/gateway/install/kubernetes/proxy.md

```diff
@@ -80,6 +80,7 @@ The control plane contains all {{ site.base_gateway }} configurations. The confi
 
 1. Create a `values-cp.yaml` file.
 
+{% if_version lte:3.4.x %}
     ```yaml
     # Do not use {{ site.kic_product_name }}
     ingressController:
```

Comment:
Not sure why Vale is triggering on codeblocks again, it shouldn't be happening :( feel free to ignore the linter here.